caasp-istio-pilot-image/caasp-istio-pilot-image.kiwi

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<!-- OBS-AddTag: caasp/v4/istio-pilot:%%PKG_VERSION%%-rev<VERSION> caasp/v4/istio-pilot:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> -->
+<!-- OBS-AddTag: caasp/v4/istio-pilot:1.5.4-->
```

Comment:
Why this change? Please use the same tagging schema of other images. Note that %%PKG_VERSION%% placeholder will be replaced by istio version if you have a _service like:
```
<service name="replace_using_package_using">
    <param name="file">caasp-istio-pilot-image.kiwi</param>
    <param name="regex">%%PKG_VERSION%%</param>
    <param name="parse-version">patch</param>
    <param name="package">istio</param>
</service>

Reply:
the image will now be reverted to use %%PKG_VERSION%% and the cilium-istio-proxy package is now being modified to build as version 1.5.4. So this hard-coded tagging can be removed and _service file can be retained.